src/main/java/mate/academy/service/UserService.java

```diff
@@ -10,11 +10,17 @@ public class UserService {
 
     /**
      * Find user by email. All users are stored in <code>private static final User[] users</code>
+     *
      * @param email - the input parameter
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
```

Comment:
let's not change the signature of the method (remove the static keyword) and simply create a variable for the UserService class in AuthenticationService